src/k8s-engine/lima.ts

```diff
@@ -1431,6 +1466,8 @@ ${ commands.join('\n') }
         if (defined(status) && status.status === 'Running') {
           await this.ssh('sudo', '/sbin/rc-service', 'k3s', 'stop');
           await this.ssh('sudo', '/sbin/rc-service', '--ifstarted', 'docker', 'stop');
+          // Always stop it, even if we're on MOBY, in case it got started for some reason.
+          await this.ssh('sudo', '/sbin/rc-service', 'buildkitd', 'stop');
```

Comment:
You'll need `--ifstarted`, otherwise it'll probably error out on moby. Same as the previous line.

Reply:
noted